pkg/client/clientset/versioned/fake/register.go

```diff
@@ -10,9 +10,11 @@ import (
 	schema "k8s.io/apimachinery/pkg/runtime/schema"
 	serializer "k8s.io/apimachinery/pkg/runtime/serializer"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	testfake "github.com/openshift/hive/pkg/test/fake"
+
 )
 
-var scheme = runtime.NewScheme()
+var scheme = testfake.GetScheme()
```

Comment:
This delta won't work, as this file will be regenerated by `make generate` (part of `make update`). We just need to focus on making sure that this clientset package isn't actually used anywhere.